src/um/wia.rs

```diff
@@ -0,0 +1,756 @@
+    ) -> HRESULT,
+    fn GetExtension(
+        lFlags: LONG,
+        bstrName: BSTR,
+        riidExtensionInterface: REFIID,
+        ppOut: *mut *mut VOID,
+    ) -> HRESULT,
+    fn GetParentItem(
+        ppIWiaItem2: *mut *mut IWiaItem2,
+    ) -> HRESULT,
+    fn GetRootItem(
+        ppIWiaItem2: *mut *mut IWiaItem2,
+    ) -> HRESULT,
+    fn GetPreviewComponent(
+        lFlags: LONG,
+        ppWiaPreview: *mut *mut IWiaPreview,
+    ) -> HRESULT,
+    fn EnumRegisterEventInfo(
+        lFlags: LONG,
+        pEventGUID: REFGUID,
+        ppIEnum: *mut *mut IEnumWIA_DEV_CAPS,
+    ) -> HRESULT,
+    fn Diagnostic(
+        ulSize: ULONG,
+        pBuffer: *const BYTE,
+    ) -> HRESULT,
+}}
+DEFINE_GUID!{IID_IWiaDevMgr2,
+    0x79C07CF1, 0xCBDD, 0x41ee, 0x8E, 0xC3, 0xF0, 0x00, 0x80, 0xCA, 0xDA, 0x7A}
+RIDL!{#[uuid(0x79c07cf1, 0xcbdd, 0x41ee, 0x8e, 0xc3, 0xf0, 0x00, 0x80, 0xca, 0xda, 0x7a)]
+interface IWiaDevMgr2(IWiaDevMgr2Vtbl): IUnknown(IUnknownVtbl) {
+    fn EnumDeviceInfo(
+        lFlags: LONG,
+        ppIEnum: *mut *mut IEnumWIA_DEV_INFO,
+    ) -> HRESULT,
+    fn CreateDevice(
+        lFlags: LONG,
+        bstrDeviceID: BSTR,
+        ppWiaItem2Root: *mut *mut IWiaItem2,
+    ) -> HRESULT,
+    fn SelectDeviceDlg(
+        hwndParent: HWND,
+        lDeviceType: LONG,
+        lFlags: LONG,
+        pbstrDeviceID: LPBSTR,
+        ppItemRoot: *mut *mut IWiaItem2,
+    ) -> HRESULT,
+    fn SelectDeviceDlgID(
+        hwndParent: HWND,
+        lDeviceType: LONG,
+        lFlags: LONG,
+        pbstrDeviceID: LPBSTR,
+    ) -> HRESULT,
+    fn RegisterEventCallbackInterface(
+        lFlags: LONG,
+        bstrDeviceID: BSTR,
+        pEventGUID: REFGUID,
+        pIWiaEventCallback: *const IWiaEventCallback,
+        pEventObject: *mut *mut IUnknown,
+    ) -> HRESULT,
+    fn RegisterEventCallbackProgram(
+        lFlags: LONG,
+        bstrDeviceID: BSTR,
+        pEventGUID: REFGUID,
+        bstrFullAppName: BSTR,
+        bstrCommandline: BSTR,
+        bstrName: BSTR,
+        bstrDescription: BSTR,
+        bstrIcon: BSTR,
+    ) -> HRESULT,
+    fn RegisterEventCallbackCLSID(
+        lFlags: LONG,
+        bstrDeviceID: BSTR,
+        pEventGUID: REFGUID,
+        pClsID: REFGUID,
+        bstrName: BSTR,
+        bstrDescription: BSTR,
+        bstrIcon: BSTR,
+    ) -> HRESULT,
+    fn GetImageDlg(
+        lFlags: LONG,
+        bstrDeviceID: BSTR,
+        hwndParent: HWND,
+        bstrFolderName: BSTR,
+        bstrFilename: BSTR,
+        plNumFiles: LPLONG,
+        ppbstrFilePaths: *mut *mut BSTR,
+        ppItem: *mut *mut IWiaItem2,
+    ) -> HRESULT,
+}}
+DEFINE_GUID!{CLSID_WiaDevMgr,
+    0xa1f4e726, 0x8cf1, 0x11d1, 0xbf, 0x92, 0x00, 0x60, 0x08, 0x1e, 0xd8, 0x11}
+DEFINE_GUID!{CLSID_WiaDevMgr2,
+    0xB6C292BC, 0x7C88, 0x41ee, 0x8B, 0x54, 0x8E, 0xC9, 0x26, 0x17, 0xE5, 0x99}
+DEFINE_GUID!{CLSID_WiaLog,
+    0xA1E75357, 0x881A, 0x419e, 0x83, 0xE2, 0xBB, 0x16, 0xDB, 0x19, 0x7C, 0x68}
```

Comment:
Can you add `RIDL!` class definitions for these three classes as well? https://github.com/retep998/winapi-rs/blob/0.3/CONTRIBUTING.md#com-classes

Reply:
This has been added

Reply:
Thanks, fixed